sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/ConnectOptions.cs

```diff
@@ -39,7 +39,7 @@ internal static class BrowserConnectOptionsConverter
 
         foreach (System.Reflection.PropertyInfo? sourceProperty in sourceType.GetProperties())
         {
-            System.Reflection.PropertyInfo targetProperty = targetType.GetProperty(sourceProperty.Name);
+            System.Reflection.PropertyInfo? targetProperty = targetType.GetProperty(sourceProperty.Name);
```

Comment:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.csproj#L11